Userland/crash.cpp

```diff
@@ -1,19 +1,65 @@
+#include <AK/Function.h>
 #include <AK/String.h>
 #include <Kernel/Syscall.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <sys/mman.h>
 
+#pragma GCC optimize("O0")
+
 static void print_usage_and_exit()
 {
-    printf("usage: crash -[sdiamfMFTtSxyX]\n");
+    printf("usage: crash -[AsdiamfMFTtSxyX]\n");
     exit(0);
 }
 
-#pragma GCC optimize("O0")
+enum CrashFailure {};
```

Comment:
Are you sure this is valid C++? What value does the `{}` (used below) generate for it?

Quick Google search turns up something about this being ill-formed; but a standard draft seems to indicate this is treated as an enum with an implicit variant...

Reply:
C++ Reference says: "If the enumerator-list is empty, the underlying type is as if the enumeration had a single enumerator with value 0)" - so I think it is fine?

I decided to change this anyway :)